srcpkgs/sof-firmware/template

```diff
@@ -22,12 +22,12 @@ do_install() {
 	for f in ${intel_path}/sof/v${version}/public-signed/*; do
 		vinstall ${f} 0644 /usr/${intel_path}/sof/public-signed
 	done
-	for arc in {bdw,byt,cht,apl,cnl,icl}; do
+	for arc in {bdw,byt,cht}; do
 		ln -s sof-${arc}-v${version}.ri ${DESTDIR}/usr/${intel_path}/sof/sof-${arc}.ri
 	done
-	ln -s sof-apl-v${version}.ri ${DESTDIR}/usr/${intel_path}/sof/sof-glk.ri
-	ln -s sof-cnl-v${version}.ri ${DESTDIR}/usr/${intel_path}/sof/sof-cfl.ri
-	ln -s sof-cnl-v${version}.ri ${DESTDIR}/usr/${intel_path}/sof/sof-cml.ri
```

Comment:
You still need to keep these three lines in the template somewhere because they create symlinks to files that don't match the original name. You should probably move them after the new for loop you created for clarity, though.

Reply:
Ah, I see. Where are those files used?

Reply:
For some reason, certain machines look for differently-named firmware than they should be looking for. For example, on my laptop, even though it should be looking for `sof-cnl.ri` (the Cannon Lake firmware), it looks for `sof-cml.ri`.